pkg/handler/handler.go

```diff
@@ -141,16 +143,29 @@ func (m *Modifier) updatePodSpec(pod *corev1.Pod, roleName, audience string) []p
 		}
 	}
 
+	hostTokenFilePath := filepath.Join(m.MountPath, m.tokenName)
+	if val, ok := pod.Spec.NodeSelector["beta.kubernetes.io/os"]; ok && val == "windows" {
+		// Convert the unix file path to a windows file path
+		// Eg. /var/run/secrets/eks.amazonaws.com/serviceaccount/token to
+		//     C:\var\run\secrets\eks.amazonaws.com\serviceaccount\token
+		absolutePath, err := filepath.Abs(hostTokenFilePath)
```

Comment:
Why the `Abs` call here? If the `MountPath` wasn't absolute to begin with,  [`filepath.Abs`](https://golang.org/pkg/path/filepath/#Abs) will join with the current working directory _of the webhook server_.

Reply:
Makes sense. Let me remove that :)